project-base/docker/conf/docker-compose-mac.yml.dist

```diff
@@ -46,23 +50,17 @@ services:
         container_name: shopsys-framework-redis-admin
         environment:
             - REDIS_1_HOST=redis
-        ports:
```

Comment:
Upgrade.md should be more descriptive. Right now I have no idea where I can work with redis admin. Also if a new records in hosts file are requested, it also should be stated in upgrade and installation guides.